src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizerPlugin.java

```diff
@@ -62,7 +62,12 @@ public MinimizerPlugin(final InstrumentingSmartRunner runner) {
             List<String> originalOrder;
             if (ORIGINAL_ORDER != null) {
                 TestPluginPlugin.info("Using specified original order. dt.minimizer.original.order argument specified: " + ORIGINAL_ORDER);
-                originalOrder = Files.readAllLines(Paths.get(ORIGINAL_ORDER));
+                Path originalOrderPath = Paths.get(ORIGINAL_ORDER);
+                originalOrder = Files.readAllLines(originalOrderPath);
+                // Copy the original order file to where we expect it to be since other parts of the tool still expects it to be there
+                // Future versions of iDFlakies should allow us to set the DetectorPathManager.originalOrderPath directly
+                Files.createDirectories(originalOrderPath.getParent());
+                Files.write(DetectorPathManager.originalOrderPath(), originalOrder);
```

Comment:
Is this fine to overwrite a potential existing original order file?

Reply:
What I can do instead is every time the original order file is detected to be different then I rename the existing one before writing. This would be more expensive but prevent overwriting something important. Let me know what you think.

Reply:
Just to be safe, I would suggest doing that backup (along with some message so user knows that the file got saved elsewhere and they can restore manually). Of course, it could end up that they keep overwriting and get more and more backups, which what you can do is to just let them know you only keep one backup (so it's on them if they forget to reset some original file), or you put some hash with the backup file name.